addons/l10n_latam_invoice_document/wizards/account_debit_note.py

```diff
@@ -5,12 +5,16 @@ class AccountDebitNote(models.TransientModel):
 
     _inherit = 'account.debit.note'
 
-    def create_debit(self):
-        """ Properly compute the latam document type of type debit note. """
-        res = super().create_debit()
-        new_move_id = res.get('res_id')
-        if new_move_id:
-            new_move = self.env['account.move'].browse(new_move_id)
-            new_move._compute_l10n_latam_document_type()
-            new_move._onchange_l10n_latam_document_type_id()
-        return res
+    def _prepare_default_values(self, move):
+        default_values = super()._prepare_default_values(move)
+
+        # properly compute debit type
+        debit_note = self.env['account.move'].new({
+            'move_type': default_values.get('move_type'),
+            'journal_id': default_values.get('journal_id'),
+            'partner_id': move.partner_id.id,
+            'company_id': move.company_id.id,
```

Comment:
If you put "debit_origin_id" here, should not it compute automatically the correct l10n_latam_identification_type_id already?